_posts/2019-08-21-rust-bloat.markdown

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title:  "Thoughts on Rust bloat"
+date:   2009-08-21 10:17:42 -0700
+categories: [rust]
+---
+I'm about to accept a [PR][druid#124] that will bloat druid's compile time about 3x and its executable size almost 2x. In this case, I think the tradeoff is worth it (without localization, a GUI toolkit is strictly a toy), but the bloat makes me unhappy and I think there is room for improvement in the Rust ecosystem.
```

Comment:
I would say 'increase' instead of 'bloat', in this intro, to be a bit less contentious. 'Bloat' is in the title so your intent is still clear, but it's a bit more delicate.